src/properties.jl

```diff
@@ -43,10 +44,15 @@ $O(3)$ as
 where $\cdot$ denotes the standard matrix-vector product.
 """
 struct EuclideanVector{T} <: AbstractProperty
-   val::SVector{3, T}
+   val::SVector{3, Complex{T}}
```

Comment:
Why complex and not real? If complex is needed then we should just allow `T` to be `Complex`. The construction can then simply be done with `EuclideanVector(ComplexF64)` (or similar...)

Reply:
I think this is now fixed.